lib/core/src/server/preview/iframe-webpack.config.js

```diff
@@ -141,6 +142,26 @@ export default async ({
             },
           ],
         },
+        {
+          test: /\.js$/,
+          use: [
+            {
+              loader: require.resolve('babel-loader'),
+              options: {
+                sourceType: 'unambiguous',
+                presets: [
+                  [
+                    require.resolve('@babel/preset-env'),
+                    { shippedProposals: true, useBuiltIns: 'usage', corejs: '3' },
+                  ],
+                ],
+                plugins,
+              },
+            },
+          ],
+          include: [/node_modules/],
+          exclude: excludePaths,
+        },
```

Comment:
this is **always** added?

Reply:
Yes, It look scary due to the include, but the exclude paths is excluding all but es6 modules.
This doesn't impact performances on startup

Reply:
I wish loader/rules could be given names or id's to tell them apart.

I think we need a comment here, telling what this loader is for.

Reply:
I can extract it to a es6-transform-loader